src/app/components/ShiftCard.tsx

```diff
@@ -4,32 +4,45 @@ import { Shift } from '../api';
 import {
   getWorkSectionColour
 } from '../api/utils';
+import { getForegroundColor } from '../infrastructure/colorUtils';
 
 export interface ShiftCardProps {
   shift: Shift;
+  title?: React.ReactNode;
 }
 
 export default class ShiftCard extends React.Component<ShiftCardProps, {}> {
 
   render() {
     const { shift } = this.props;
     const { startDateTime } = shift;
-    const startTime = moment.isMoment(startDateTime) ? startDateTime.format('HH:MM') : startDateTime;
+    const startTime = moment(startDateTime).format('HH:MM');
```

Comment:
Should this be format('HH:mm') instead of 'HH:MM'?

Reply:
good question, I just used whatever was there before.  
Looks like MM is month instead of time.  hence why everything is showing :03 lol.  will change to mm